sites/docs/docs/features/queries/sql-queries.md

```diff
@@ -5,15 +5,16 @@ hide_table_of_contents: false
 title: Writing Queries
 ---
 
-When you open a page in dev mode, Evidence runs all of the queries on the page. You can see the progress of these queries printed in the console. In dev mode, Evidence monitors the contents of your SQL blocks, and reloads the page as necessary to reflect any changes you've made to your queries. 
+When you open a page in dev mode, Evidence runs all of the queries on the page. You can see the progress of these queries printed in the console. In dev mode, Evidence monitors the contents of your SQL blocks, and reloads the page as necessary to reflect any changes you've made to your queries.
 
 ## Query Blocks
 You can include SQL queries in your page using a markdown code block (starting and ending with 3 backticks). Evidence requires a query name to be supplied directly after the first 3 backticks.
 
 ````markdown
 ```sales_by_country
 select country, sum(sales) as sales
-from international_transactions 
+from international_transactions
+group by 1
```

Comment:
Good catch. Some SQL dialects (I believe eg Snowflake?) would run this but many would choke